modules/template/elements/BaseContentRecordElementVariable.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/**
+ * @link https://www.humhub.org/
+ * @copyright Copyright (c) HumHub GmbH & Co. KG
+ * @license https://www.humhub.com/licences
+ */
+
+namespace humhub\modules\custom_pages\modules\template\elements;
+
+use humhub\modules\content\components\ContentActiveRecord;
+use yii\db\ActiveRecord;
+
+class BaseContentRecordElementVariable extends BaseRecordElementVariable
+{
+    public ?UserElementVariable $author;
+
+    public function setRecord(?ActiveRecord $record): self
+    {
+        parent::setRecord($record);
+
+        if ($this->record instanceof ContentActiveRecord) {
+            $this->author = UserElementVariable::instance($this->elementContent)
```

Comment:
@yurabakhtin Would it be beneficial to introduce a `getAuthor` here so that we only have to load the author record when it is actually needed?

Reply:
@luke- Yes, good idea, done https://github.com/humhub/custom-pages/pull/414/commits/205871a8aea0107cadd0033d6a555e50b84190ed.